### ISSUE TITLE
replace DESCRIBE FIELD with RTTI?

### ISSUE THREAD

**Issue by larshp** [sbcgua/ajson#88]
as the statement does not work in Steampunk,

```
./src/zcl_ajson.clas.locals_imp.abap[604, 9]  - Statement does not exist in ABAPCloud(or a parser error), "describe" (parser_error) [E]
./src/zcl_ajson.clas.locals_imp.abap[698, 7]  - Statement does not exist in ABAPCloud(or a parser error), "describe" (parser_error) [E]
```

**Comment by sbcgua** [comment]
will be by-the-way solved by#89

**Comment by sbcgua** [comment]
can be closed ? @larshp
